modules/installation-guide/examples/che-proc_building-an-offline-plug-in-registry-image.adoc

```diff
@@ -0,0 +1,43 @@
+[id="building-an-offline-plug-in-registry-image_{context}"]
+= Building an offline plug-in registry image
+
+This section describes how to build an _offline_ plug-in registry image. Starting workspaces without relying on resources from the outside Internet requires building this image. The image contains plug-in metadata and all plug-in or extension artifacts. 
+
+.Procedure
+
+. Clone the plug-in registry repository:
++
+----
+$ git clone https://github.com/eclipse/che-plugin-registry.git
+$ cd che-plugin-registry
+----
+
+. Check out version of the registry to deploy:
++
+[subs="+attributes,+quotes"]
+----
+$ git checkout {prod-ver}.x
+----
+
+. Build the plug-in registry image, including only the _latest_ version of each plug-in:
++
+[subs="+quotes"]
+----
+./build.sh --organization _<my-org>_ \
+           --registry _<my-registry>_ \
+           --tag _<my-tag>_ \
+           --offline \
+           --latest-only
+----
++
+[TIP]
+====
+* To display full options for the `build.sh` script, use the `--help` parameter.
+
+* To build a container image containing all possible versions of the `.vsix` and `.theia` extensions, omit the `--latest-only` argument. This results in a large image.
```

Comment:
I mean, yes, it's true. But do we want to call it out?

Reply:
So just remove the complete line ? 
```suggestion
```